runtime/port/CMakeLists.txt

```diff
@@ -31,33 +31,57 @@
 
 
 set(OBJECTS "")
-list(APPEND OBJECTS 
+list(APPEND OBJECTS
+	j9gp.c
```

Comment:
Can these lists be kept in alphabetical order?

Reply:
Sure, I just reordered so that it would match the module.xml. I'm assuming I should reorder that as well?